pkg/agent/baker.go

```diff
@@ -438,7 +438,7 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 		"AnyAgentIsLinux": func() bool {
 			return cs.Properties.AnyAgentIsLinux()
 		},
-		"IsNSeriesSKU": func(profile *datamodel.AgentPoolProfile) bool {
+		"IsNSeriesSKU": func() bool {
 			return config.EnableNvidia
 		},
```

Comment:
the function should return if this SKU is N series. The function is doing another thing. will it possible that EnableNvidia != NSeriesSKU?

Reply:
No EnableNvidia is determined in RP https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp?path=%2Fresourceprovider%2Fserver%2Fmicrosoft.com%2Fcommon%2Fagentbaker%2Fagentbakerhelper.go&version=GBmaster&line=88&lineEnd=88&lineStartColumn=3&lineEndColumn=80&lineStyle=plain&_a=contents
where it's compared case insensitive

Reply:
yea my concern is a nit that the naming of the function does not exactly matching what it says. functionality looks good to me.